packages/react/src/components/cookieConsent/languageSwitcher/LanguageSwitcherItem/LanguageSwitcherItem.module.scss

```diff
@@ -158,7 +153,7 @@
   .dropdownItem.active:after,
   .dropdownLinkItem:hover:not(:focus-within):after,
   .dropdownLinkItem.active:after {
-    background-color: var(--item-active-color);
+    background-color: var(--color-black-90);
```

Comment:
Same theming comment here as previously?

Reply:
These are previously inherited from deprecated Navigation component and no longer would have effect here. Thus root level variables 🤔